src/main/java/edu/kit/ipd/crowdcontrol/objectservice/moneytransfer/AmazonMailFormatChangedException.java

```diff
@@ -0,0 +1,12 @@
+package edu.kit.ipd.crowdcontrol.objectservice.moneytransfer;
+
+/**
+ * An exception, that gets thrown, if the parser cannot parse the mails, because the mail format changed.
+ * @author Felix Rittler
+ */
+public class AmazonMailFormatChangedException extends Exception{
+
+    public AmazonMailFormatChangedException() {
+        super("The Parser cannot extract the giftcode from the mails, because the mail format changed. You need to adjust the parser to the new mail format.");
+    }
+}
```

Comment:
Well if I get a amazon giftcard why not just caching the html text and send them again to the worker ? without parsing out a giftcode ? as far as I can see the cards are having a fancy link where you click on - log into your amazon acc. - and you got more money. Why parsing such a mail?